server/index.ts

```diff
@@ -194,6 +227,17 @@ router.post('/message', async (req, res) => {
     version: '2'
   });
 
+  if (msg.type === 'delete-proposal') {
+    let query = `SELECT address FROM messages WHERE type = 'proposal' AND id = ?`;
+    let propasalSigner = await db.queryAsync(query, [msg.payload.proposal]);
+    if (
+      propasalSigner[0].address.toLowerCase() !== body.address.toLowerCase()
```

Comment:
The address is never lowercase, you could update the query to use `WHERE address = ?` to tackle this

Reply:
I couldn't get that to work with `WHERE address = ?` it gives me undefined error that I can't manage to fix. I've removed the `toLowerCase()` now, I'm not sure anymore why I needed them before.

Reply:
Hum that's weird